Makefile

```diff
@@ -156,7 +156,7 @@ test-unit:
 	CURATOR_IMAGE=quay.io/openshift/origin-logging-curator:latest \
 	FLUENTD_IMAGE=$(FLUENTD_IMAGE) \
 	go test -cover -race ./pkg/...
-	go test -cover -race ./test/helpers/
+	go test -cover -race ./test/helpers/...
```

Comment:
@vimalk78 thinking a bout this, does it make more sense to leave these helper tests in there original packages and just move the test execution to be part of functional?  Maybe we make `test-unit` only compile them for sanity but not run?

Reply:
we could say that the tests in `./test/helpers` are only for helpers required in `e2e` or `functional` tests. the `make` default target already compiles the e2e tests for sanity, we could compile these also there.

Reply:
updated PR as per suggestion